src/main/java/com/datapipe/jenkins/vault/jcasc/secrets/VaultSecretSource.java

```diff
@@ -227,8 +227,8 @@ private Optional<String> getVariable(String key) {
         return Optional.ofNullable(prop.getProperty(key, System.getenv(key)));
     }
 
-    private Optional<String[]> getCommaSeparatedVariables(String key) {
-        return getVariable(key).map(str -> str.split(","));
+    private Optional<String[]> getSeparatedVariables(String key) {
+        return getVariable(key).map(str -> str.split("\\s*(\\s|,)\\s*"));
```

Comment:
Think `,\s?` Will satisfy most use cases for splitting.
The current regex will break paths with spaces.